starlite/storage/memcached_backend.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from aiomcache import Client
+
+from .base import StorageBackend
+
+
+class MemcachedStorageBackend(StorageBackend):
+    __slots__ = ("_memcached",)
+
+    def __init__(self, memcached: Client, key_prefix: str | None = None) -> None:
+        super().__init__(key_prefix=key_prefix)
+        self._memcached = memcached
+
+    @classmethod
+    def with_client(
+        cls,
+        host: str,
+        *,
+        port: int = 11211,
```

Comment:
use a constant for this value at the top of the file?

Reply:
Why? This way it's more transparent in the docs and for IDEs because the actual default value will be displayed.